docs/2.build/4.web3-apps/integrate-contracts.md

```diff
@@ -57,7 +57,7 @@ To create a wallet, simply import the `Wallet` object from the module and initia
 
 Under the hood (check the `near` tab) you can see that we are actually setting up the wallet selector, and asking it if the user logged-in already. During the setup, we pass a hook to the wallet selector, which will be called each time a user logs in or out.
 
-<details markdown="1">
+<details markdown="1" id="setting-customs-rpc-endpoints">
```

Comment:
this `id` doesn't look to be used anywhere.

Reply:
It's used in "If you want to use a custom RPC provider with NEAR Wallet Selector, [check this example](https://docs.near.org/build/web3-apps/integrate-contracts#setting-customs-rpc-endpoints)." in https://docs.near.org/api/rpc/providers (not changed in this pr), but when you click the link, it doesn't scroll to the example because the id didn't exist